### ISSUE TITLE
Railtie fails to load causing rails to fail at startup

### ISSUE THREAD

**Issue by jocubeit** [jonhue/avatari#1]
Rails (5.2.0.beta2) project fails to load because of constant error.

```
/app/vendor/bundle/ruby/2.4.0/gems/bootsnap-1.1.5/lib/bootsnap/load_path_cache/core_ext/active_support.rb:49:in `block in load_missing_constant': uninitialized constant Avatari::Railtie::Mozaic (NameError)
```

This occurs because the Mozaic constant isn't required prior to calling the `initializer 'avatari.mozaic'` block in the Railtie. To get around this I added the mozaic gem to my Gemfile before the avatari gem. 

Ideally however, you could add a require statement in the Railtie and maybe prefix the constant with a `::`.

**Comment by jonhue** [comment]
@JurgenJocubeit

Valid point. I'll fix this in the next version.

**Comment by jonhue** [comment]
Published 2.0.2
